src/coreclr/tools/Common/TypeSystem/Common/SignatureVariable.cs

```diff
@@ -66,6 +66,10 @@ protected override TypeFlags ComputeTypeFlags(TypeFlags mask)
                 flags |= TypeFlags.SignatureTypeVariable;
             }
 
+            flags |= TypeFlags.HasGenericVarianceComputed;
+            flags |= TypeFlags.HasFinalizerComputed;
+            flags |= TypeFlags.AttributeCacheComputed;
```

Comment:
Unfortunately, this work operates almost entirely over uninstantiated types and methods, and has a few spots where it actually needs to use these things. However, it might be the case that I can only set a minimal number of flags. I enabled them all as it seemed like the right thing to do, but I believe the only 1 I really needed to work was the attribute cache.

Reply:
What's the stack that leads to this being needed? This really feels like something where the only right answer is to unask the question.